extension/src/popup/components/manageAssets/ManageAssetRowButton/index.tsx

```diff
@@ -129,9 +129,6 @@ export const ManageAssetRowButton = ({
         }),
```

Comment:
these changes do a better job of showing a loading state on the asset row's button while Blockaid is scanning the asset